sample/README.md

```diff
@@ -1,24 +1,21 @@
 # Sample System
 
-This folder contains a sample implementation of a recommender system.
+This folder contains a sample implementation of a baseline recommender system that scores articles based on user topics.
+
 
 ## Overview
 
 The system has several stages of execution:
-- First makes sure an Elasticsearch index with the correct mappings exist. If not it will be created.
-- Then it queries the arXivDigest API for articles to add to the index. These are the articles that will be candidate recommendations later.
-- Next the system queries the arXivDigest API for user information. This is the information which the system bases its personalized recommendations on.
-- The system then creates personalized recommendations for each user, currently the recommendations are created by searching the Elasticsearch index with the users keywords for the best matching articles. 
-- Finally the articles found for each user are submitted to the arXivDigest API as recommendations. 
-
-## Usage
+  * First, it makes sure an Elasticsearch index with the correct mappings exist.  If not it will be created.
+  * Then, it queries the arXivDigest API for articles to add to the index.  These are the articles that will serve as candidate recommendations later.
+  * Next, the system queries the arXivDigest API for user information.  This is the information that the system bases its personalized recommendations on.
+  * The system then creates personalized recommendations for each user.  Specifically, it searches the Elasticsearch index for the best matching articlesm using the user's topics as queries.  Then, it aggregates, for each article, the relevance scores of all topics.
```

Comment:
it searches the Elasticsearch index for the best matching **articlesm** using the user's

Typo

Reply:
Fixed